test/k8s-integration/driver-config.go

```diff
@@ -82,22 +81,32 @@ func generateDriverConfigFile(platform, pkgDir, storageClassFile, snapshotClassF
 	   dataSource
 	*/
 
-	// TODO: Support adding/removing capabilities based on Kubernetes version.
-	switch deploymentStrat {
+	switch testParams.deploymentStrategy {
 	case "gke":
-		fallthrough
-	case "gce":
 		caps = append(caps, "controllerExpansion", "nodeExpansion")
+		if *imageType == "cos" {
```

Comment:
imageType can be a part of testParams too?

Reply:
Done